nx-X11/programs/Xserver/hw/nxagent/NXextension.c

```diff
@@ -296,7 +296,8 @@ CloseDownExtensions()
 
     for (i = NumExtensions - 1; i >= 0; i--)
     {
-	(* extensions[i]->CloseDown)(extensions[i]);
+	if (extensions[i]->CloseDown)
+	    (* extensions[i]->CloseDown)(extensions[i]);
```

Comment:
Hi Uli,

On Sun Jun 19 22:34:01 2016 GMT+0200, Ulrich Sibiller wrote:
> In nx-X11/programs/Xserver/hw/nxagent/NXextension.c:
> > @@ -296,7 +296,8 @@ CloseDownExtensions()
> >  
> >      for (i = NumExtensions - 1; i >= 0; i--)
> >      {
> > -	(* extensions[i]->CloseDown)(extensions[i]);
> > +	if (extensions[i]->CloseDown)
> > +	    (* extensions[i]->CloseDown)(extensions[i]);
> Hmm, same code as in dix/extensions.c. Can't this be simplified somehow? 
> —

This currently work in progress. See PR #120.
-- 
Sent from my Jolla